plat/xilinx/zynqmp/sip_svc_setup.c

```diff
@@ -39,6 +43,9 @@ static int32_t sip_svc_setup(void)
 	/* PM implementation as SiP Service */
 	pm_setup();
 
+	/* IPI implementation as SiP Service */
+	ipi_mb_setup();
```

Comment:
The patches in my branch are in order, however it ends up different order in the pull request.
I will be more careful to when I create another pull request based on other comments.

Reply:
Hmm, that's odd - something is going wrong when you're creating the PR. Confirm that you're following the steps in the contributing guidelines.
NOTE - for the updates, you should just update the branch on your github fork of ARM TF, and github will pick up the changes in this PR; no need to create a new one.

Reply:
There's nothing wrong with the patch order in the PR. It's just that the GitHub UI unfortunately shows patches in order of when they were authored, instead of commit order, which is really annoying.